experiments/pyexps/farmem.py

```diff
@@ -23,6 +23,7 @@
 import simbricks.orchestration.experiments as exp
 import simbricks.orchestration.nodeconfig as nodec
 import simbricks.orchestration.simulators as sim
+from simbricks.orchestration.experiment.experiment_environment import ExpEnv
```

Comment:
Please never just import one class or function from a module. Instead always import the whole module. We use Google's Python style guide. See [here](https://google.github.io/styleguide/pyguide.html#22-imports) for more info on this.

I know that we currently still violate this in multiple places but I hope to resolve this some time soon.

Reply:
I also just noticed this with #132. Maybe you can add a commit here to fix it for #132 as well?